packages/react/src/component-tests/IcAccordion/IcAccordion.cy.tsx

```diff
@@ -0,0 +1,334 @@
+/* eslint-disable react/jsx-no-bind */
+/// <reference types="Cypress" />
+
+import React from "react";
+import { mount } from "cypress/react";
+import {
+  AccordionsWithDisabled,
+  GroupWithOneExpanded,
+  SimpleAccordion,
+  SimpleExpandedAccordion,
+  TwoAccordions,
+  TwoAccordionsWithOneExpanded,
+} from "./IcAccordionTestData";
+import { IcAccordion, IcAccordionGroup } from "../../components";
+import { BE_VISIBLE, CONTAIN_TEXT } from "../utils/constants";
+import { CYPRESS_AXE_OPTIONS } from "../../../cypress/utils/a11y";
+import { SlottedSVG } from "../../react-component-lib/slottedSVG";
+
+const DEFAULT_TEST_THRESHOLD = 0.03;
```

Comment:
On my windows machine I get consistent failures on three tests due to threshold issues:
![Screenshot 2024-01-16 at 08 58 56](https://github.com/mi6/ic-ui-kit/assets/111292477/1c1ae589-7bc8-4d5c-b95a-792fdd1b1757)
![Screenshot 2024-01-16 at 09 27 58](https://github.com/mi6/ic-ui-kit/assets/111292477/4c4aa468-497b-4d8e-813d-68f77cd40926)

I've found that if you increase the threshold to 0.05 for those tests, they pass :)